web/privacy.html

```diff
@@ -0,0 +1,521 @@
+    Park, CA 94025, United States. If a person lives outside of the United
+    States or Canada, the controller is the Facebook Ireland Ltd., 4 Grand
+    Canal Square, Grand Canal Harbour, Dublin 2, Ireland.</p>
+
+<p>With each call-up to one of the individual pages of this Internet website,
+    which is operated by the controller and into which a Facebook component
+    (Facebook plug-ins) was integrated, the web browser on the information
+    technology system of the data subject is automatically prompted to download
+    display of the corresponding Facebook component from Facebook through
+    the Facebook component. An overview of all the Facebook Plug-ins may be
+    accessed under https://developers.facebook.com/docs/plugins/. During the
+    course of this technical procedure, Facebook is made aware of what
+    specific sub-site of our website was visited by the data subject.</p>
+
+<p>If the data subject is logged in at the same time on Facebook, Facebook
+    detects with every call-up to our website by the data subject—and for the
+    entire duration of their stay on our Internet site—which specific sub-site
+    of our Internet page was visited by the data subject. This information is
+    collected through the Facebook component and associated with the respective
+    Facebook account of the data subject. If the data subject clicks on one of
+    the Facebook buttons integrated into our website, e.g. the "Like" button,
+    or if the data subject submits a comment, then Facebook matches this
+    information with the personal Facebook user account of the data subject and
+    stores the personal data.</p>
+
+<p>Facebook always receives, through the Facebook component, information about
+    a visit to our website by the data subject, whenever the data subject is
+    logged in at the same time on Facebook during the time of the call-up to
+    our website. This occurs regardless of whether the data subject clicks on
+    the Facebook component or not. If such a transmission of information to
+    Facebook is not desirable for the data subject, then he or she may prevent
+    this by logging off from their Facebook account before a call-up to our
+    website is made.</p>
+
+<p>The data protection guideline published by Facebook, which is available at
+    https://facebook.com/about/privacy/, provides information about the
+    collection, processing and use of personal data by Facebook. In addition,
+    it is explained there what setting options Facebook offers to protect the
+    privacy of the data subject. In addition, different configuration options
+    are made available to allow the elimination of data transmission to
+    Facebook. These applications may be used by the data subject to eliminate
+    a data transmission to Facebook.</p>
+
+<h4>10. Data protection provisions about the application and use of Twitter</h4>
+<p>On this website, the controller has integrated components of Twitter.
```

Comment:
We don'T have a "share on twitter" button, only the possiblity to log in using twitter.